.github/workflows/build.yml

```diff
@@ -152,45 +152,81 @@ jobs:
           ls -alh
           cd code/mobile/android/PhoneVR
           chmod +x prepare-alvr-deps.sh
-          bash prepare-alvr-deps.sh
+          bash prepare-alvr-deps.sh nogvr
```

Comment:
prepare-alvr-deps.sh takes lot of time. Cant it be done that while making nogvr flavour gradle picks up the gvr libs only when building gvr varient ? 
prepare-alvr-deps.sh should only be run once, it will be huge burdan to test gvr and noGvr flavrous in development if we need to run prepare-alvr-deps.sh for everytime we need to test gvr noGvr flavour

Reply:
That already works, the CI does it the other way to ensure there are no accidential dependencies on gvr sdk

Reply:
Oh but it does include GVR .so file in noGvr build if prepared with gvr libs, I should probably fix that

Reply:
Fixed